css/includes/_base.scss

```diff
@@ -346,6 +362,11 @@ table {
     font-size: 0.7rem;
 }
 
+.badge {
+    background: var(--glpi-badge-bg);
+    color: var(--glpi-badge-fg) !important;
```

Comment:
I'm not sure about the !important here.
It breaks color in ticket headers with light palettes
![image](https://github.com/user-attachments/assets/12473a07-7622-45b7-b5e4-6741e83c8847)

It's also visible in kanban
![image](https://github.com/user-attachments/assets/d2117930-435a-4676-b307-c7e982dba302)